### ISSUE TITLE
Java Application Run Configuration "Show Command Line" button text display exceed its button border

### ISSUE THREAD

**Issue by caldy2000** [eclipse-jdt/eclipse.jdt.debug#462]
# Operation environment
- win11 pro workstation
- eclipse 2024-06
- openjdk 9
- my locale

    zh_CN

- my preference
  - dialog font
    
    font name: `source code pro`

    font size: 11
  - display resolution

    `1920x1080`, intel display default settings.
# Case
A Java class which has main method, check its run configuration plane, the button `Show Command Line` locates at bottom right corner,its button text display exceed the button border, but the exceed part is unvisible.

screen shot as follows:
![image](https://github.com/user-attachments/assets/191f9e4d-9828-4636-8e88-221ce3c373df)

# Try

1. restore dialog font to default(Use System Font)

    by doing this,this button display correctly.

2. set font size greater than 9(default size)

    Use system font, and when size greater than 9, it is going to exceed the button border like what i described before.

I dont know if it works well on your operation environment.

**Comment by SougandhS** [comment]
Hi @caldy2000 
Is this still happening for [latest version](https://eclipseide.org/) ?

**Comment by caldy2000** [comment]
@SougandhS YES, 26-06 R same. Microsoft offers "Microsoft YaHei UI" font as default for chinese, and tried set to" segoe ui ", regular, size 12, like this 

<img width="947" height="624" alt="Image" src="https://github.com/user-attachments/assets/846276e1-61e0-4974-9762-a1e5c42ccde1" />

<img width="1000" height="686" alt="Image" src="https://github.com/user-attachments/assets/066a282e-41d3-44fd-9b70-e5d9d7136fc8" />

you could see the "show command line" button.
also, everytime start Eclipse, the Windows Defender dialog from Eclipse have part of text hide in vertical direction, see this :

<img width="1262" height="554" alt="Image" src="https://github.com/user-attachments/assets/ae9d2090-0639-4f65-867c-56bb015c7746" />

**Comment by caldy2000** [comment]
@SougandhS continue the context, "In general adding exclusions ....." statement, the 'g', 'y' character‘s bottom part

**Comment by SougandhS** [comment]
Thanks for checking with the latest version, I will try to check with the given environment. 👍

**Comment by deepika-u** [comment]
Able to see the problem on the below environment 
```
Eclipse SDK
Version: 2026-09 (4.41)
Build id: I20260629-2300
OS: Windows 11, v.10.0, x86_64 / win32
Java vendor: Oracle Corporation
Java runtime version: 26+35-2893
Java version: 26
```

Setting from preferences
<img width="1576" height="897" alt="Image" src="https://github.com/user-attachments/assets/089ff44e-1078-4608-86c0-ca448d66d29f" />

Screen 1 in "debug configuration"
<img width="1147" height="1008" alt="Image" src="https://github.com/user-attachments/assets/e9877e7e-4b46-467a-b8c3-0c024ba71228" />

Screen 2 in "run configuration"
<img width="1362" height="1010" alt="Image" src="https://github.com/user-attachments/assets/12c8c337-f2ab-45ec-b68f-f1aeb99ae092" />

**Comment by SougandhS** [comment]
Fixed via https://github.com/eclipse-platform/eclipse.platform/pull/2788
